cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -153,6 +170,61 @@ function ManagementCredentials(props: Partial<StepProps>) {
         }
     };
 
+    useEffect(() => {
+        if (awsState[STORE_SECTION_FORM].REGION) {
+            retrieveOsImages(awsState[STORE_SECTION_FORM].REGION);
+        }
+    }, [awsState[STORE_SECTION_FORM].REGION]);
+
+    function retrieveOsImages(region: string) {
+        try {
+            setOsImages([]);
+            if (region !== '') {
+                AwsService.getAwsosImages(region).then((data) => {
+                    setOsImages(data);
+                    setErrorMessage((errorMessage) => {
+                        const copy = { ...errorMessage };
+                        delete copy['OS_IMAGE'];
```

Comment:
Can we use the AWS FIELDS enum here instead of OS_IMAGE string?

Reply:
Yes sir. Changed it to `AWS_FIELDS.OS_IMAGE`